README.md

```diff
@@ -81,17 +85,30 @@ The following are examples of valid sentences:
 Write the BNF (not EBNF) description for this language.
 
 ## A2.1
-
-«replace this with your answer»
+```
+<sentence>    ::= "The" <adjectives> <noun> <verb> <adverb> "."  
+  
+<adjectives>  ::= <adjective> <adjective>  
+<adjective>   ::= <empty> | "lazy" | "smelly"  
+<noun>        ::= "dog" | "cat"  
+<verb>        ::= "ate" | "ran"  
+<adverb>      ::= <empty> | "slowly" | "noisily"  
+```
 
 
```

Comment:
# 7

I'm giving you the benefit of the doubt: I assume line 91 contains a typo, and the first entry of the RHS was meant to be plural,

Even then, I think it's preferable to state it as:

~~~
<adjectives>  ::= <adjectives> <adjective>  | «empty»
<adjective>   ::=  "lazy" | "smelly" 
~~~